### ISSUE TITLE
Grey color on green background is hardly readable

### ISSUE THREAD

**Issue by hbruch** [stadtnavi/stadtnavi_app#28]
**Describe the bug**
Grey color on green background is hardly readable

**Expected behavior**
Text/Symbols should have a contrast from background

**Screenshots**
<img width="343" alt="image" src="https://user-images.githubusercontent.com/2187389/117274085-01c7e080-ae5d-11eb-848c-49d45683bb87.png">

**Additional context**
Instead of using grey color, using e.g. reduced opacity might work no matter how the app is branded

**Comment by SamuelRioTz** [comment]
![WhatsApp Image 2021-05-12 at 01 03 06](https://user-images.githubusercontent.com/43000324/117920959-e413d480-b2bd-11eb-8078-f4318da01dff.jpeg)
temporary style until new design decisions
